web/docs/language/features.md

```diff
@@ -686,9 +688,13 @@ app MyApp {
 #### `userEntity: entity` (required)
 Entity which represents the user (sometimes also referred to as *Principal*).
 
-#### `methods: [AuthMethod]` (required)
+#### `externalAuthAssociationEntity: entity` (optional)
+Entity which associates a user with some external authentication provider. We currently offer support for [Google](#google).
```

Comment:
Aha, so they pick an entity they want to use? What would it typically be, can it be User again?

Reply:
Yeah, it follows the same convention we use for User. It cannot be User itself, as it needs a many-to-one relationship to User. I don't think we would want to do it under the covers since if we add/remove an entity as they enable/disable it, they will potentially lose the table data when they disabled it (possibly by accident). I think keeping it explicit is the way to go, and they can decorate it with additional fields/info.